lib/encoder.py

```diff
@@ -836,29 +857,33 @@ def StoreEncoding(self, encoding, workdir=None):
   def ReadEncodingResult(self, encoding, scoredir=None):
     """Reads an encoding result back from storage, if present.
 
-    None is returned file does not exist."""
+    None is returned if file does not exist.
```

Comment:
The search path can be more than one directory. We have two use cases:
- We know exactly which directory we want to read from (used in export_improved_scores and verify_scores)
- We don't know exactly which directory we want to read from, and must use the configured scorepath (used most other places).

This was as simple as I found a way to make it.